assets/questions/rle.js

```diff
@@ -26,26 +26,33 @@ globalData.questions['rle'] = {
   },
   auxiliaryCode: {
     python:
-`def skipEncodingAtEndOfString(word):
-    pass
+`class AuxiliaryCode(object):
+    @classmethod
+    def skipEncodingAtEndOfString(cls, word):
+        pass
 
-def ignoreStringLengthWhenEncoding(word):
-    pass
+    @classmethod
+    def ignoreStringLengthWhenEncoding(cls, word):
+        pass
 
-def failToDemarcateBeginningOfEncodedChunk(word):
-    pass
 
-def DecodeEncodedString(word):
-    pass
+    @classmethod
+    def failToDemarcateBeginningOfEncodedChunk(cls, word):
+        pass
+
+
+    @classmethod
+    def DecodeEncodedString(cls, word):
+        pass
 `
   },
   prompts: [{
     instructions: [
       [
-        'Implement the encode function. It takes a string as input and ',
-        'returns an encoding of the string where long runs of characters are ',
-        'replaced by <# characters>x<character>. For example, "abcccccd" ',
-        'should be encoded as "ab5xc".'
+        'In this question, you\'ll implement the encode function. It takes a ',
+        'string as input and returns an encoding of the string where long ',
+        'runs of characters are replaced by <# characters>x<character>. For ',
```

Comment:
Btw just wondering -- in this question is the original string composed only of alphabetic characters? I ask because if numbers are part of the original string, it gets tricky.